tools/bash_completion

```diff
@@ -0,0 +1,43 @@
+__gohan()
+{
+
+
+    declare -A possibleWord=( ["1"]="gohan"
+    ["2"]="--debug --help --version -d -h -v"
+    ["3"]="client validate v init-db idb convert conv server srv test_extensions test_ex migrate mig template run test test openapi markdown dot glace-server gsrv help h")
```

Comment:
This task has been done in this commit.
https://github.com/cloudwan/gohan/pull/487/commits/e775c76883c38c22d93467955055deafea0da2a3